src/handlers/itemStore.ts

```diff
@@ -121,5 +121,90 @@ export const itemStoreHandlers: NatsHandler[] = [
                 }
             }
         }
-    ]
+    ],
+    [
+        "item_store_inventory_add_item",
+        async (subscription: Subscription): Promise<void> => {
+            const client = getClient();
+
+            for await (const message of subscription) {
+                const inventory_item = jsonCodec.decode(message.data) as JSONInventoryItem;
+
+                try {
+                    const reponse = await client.index({
+                        index: ELASTICSEARCH_INVENTORY_INDEX_NAME, 
+                        id: String(inventory_item.user_id),
+                        body: {
+                            doc : inventory_item.instance_number
+                        }
+                    });
+
+                    console.log(
+                        `[INVENTORY] Item added to ${ELASTICSEARCH_INDEX_NAME}`,
```

Comment:
the name in the brackets should still be `ITEM-STORE` instead of Inventory, it's actually the name of the process and it will help us filter the logs when they're stored in a central location